examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,25 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points.
```

Comment:
```suggestion
The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of
points. Here, we supply the data as :class:`numpy.ndarray` with the ``x``, ``y``,
and ``z`` parameters and draw the contours using a 0.5p pen.
```
I have suggested adding a second sentence with a bit more detail about the method.